boot/bootutil/src/loader.c

```diff
@@ -182,6 +182,11 @@ fill_rsp(struct boot_loader_state *state, struct boot_rsp *rsp)
             break;
         }
     }
+    /* At least one image must be active, otherwise skip the execution */
+    if(BOOT_CURR_IMG(state) >= BOOT_IMAGE_NUMBER)
```

Comment:
Should be a space between the 'if' and the '('.

Reply:
Fixed